src/components/calcite-color-picker/calcite-color-picker.e2e.ts

```diff
@@ -799,15 +799,15 @@ describe("calcite-color-picker", () => {
 
           await rgbModeButton.click();
 
-          expect(await rInput.getProperty("value")).toBeNull();
-          expect(await gInput.getProperty("value")).toBeNull();
-          expect(await bInput.getProperty("value")).toBeNull();
+          expect(await rInput.getProperty("value")).toBeFalsy();
```

Comment:
Can you assert against the explicitly expected type? Currently, it would allow `false`, `0`, `''`, `null`, `undefined`, and `NaN`.